bin/sdk_gen

```diff
@@ -71,31 +71,6 @@ const run = (command) => {
   return result
 }
 
-/**
- * Updates release-please-config.json to stamp the correct CalVer for the SDKs
- *
- * if `release-as` is already set, CalVer patch will be bumped if the current
```

Comment:
@jkaster I've removed this function becase we don't need to set release-as anymore with my changes. However i'm confused by this statement.`if `release-as` is already set, CalVer patch will be bumped if the current version is the same as the requested release version`. When will this happen? Afaict, when we are generating new bindings, the repo should not be on the new calendar version.

Reply:
Sometimes we release, e.g., 23.0.0 then later 23.0.1. The `sdk_gen` script uses the current main `release-as` value to bump the patch version if `release-as` is not blank.